op-e2e/actions/helpers/action.go

```diff
@@ -81,8 +83,9 @@ func (st *defaultTesting) Ctx() context.Context {
 // The format and args behave the same as fmt.Sprintf, testing.T.Errorf, etc.
 func (st *defaultTesting) InvalidAction(format string, args ...any) {
 	st.TestingBase.Helper() // report the error on the call-site to make debugging clear, not here.
-	st.Errorf("invalid action err: "+format, args...)
+	assert.Fail(st, fmt.Sprintf("invalid action err: "+format, args...))
```

Comment:
Improves error logging, so that the error is easily visible in action test logs and the test exits immediately so the error log can be found towards the bottom of the log.

Reply:
What I understand is that the original intent of `InvalidAction` is to not stop the test
https://github.com/ethereum-optimism/optimism/blob/3299a3cb0cbabc7c6d070e32c96701bd1c586fbc/op-e2e/actions/helpers/action.go#L15-L21
https://github.com/ethereum-optimism/optimism/blob/3299a3cb0cbabc7c6d070e32c96701bd1c586fbc/op-e2e/actions/helpers/action.go#L80

Reply:
Going to remove that comment since it was out of date.

Reply:
I see that the comment embedded at ` type Testing interface` should be updated.

Reply:
Updated.